Description: Put all orphan files into the airporthostmon package
 Some files were not declared as being part of a package.
Author: Julien BLACHE <jblache@debian.org>

Index: airport-utils-2/hostmon/AboutDialog.java
===================================================================
--- airport-utils-2.orig/hostmon/AboutDialog.java	2010-03-11 20:55:55.891754138 +0100
+++ airport-utils-2/hostmon/AboutDialog.java	2010-03-11 20:56:29.765753267 +0100
@@ -20,7 +20,7 @@
  */
 
 
-
+package airporthostmon;
 
 import java.awt.*;
 import java.awt.event.*;
Index: airport-utils-2/hostmon/AirportDiscoverer.java
===================================================================
--- airport-utils-2.orig/hostmon/AirportDiscoverer.java	2010-03-11 20:55:55.900753644 +0100
+++ airport-utils-2/hostmon/AirportDiscoverer.java	2010-03-11 20:56:29.778901828 +0100
@@ -20,7 +20,7 @@
  */
 
 
-
+package airporthostmon;
 
 
 
Index: airport-utils-2/hostmon/AirportDiscoveryInfo.java
===================================================================
--- airport-utils-2.orig/hostmon/AirportDiscoveryInfo.java	2010-03-11 20:55:55.908754228 +0100
+++ airport-utils-2/hostmon/AirportDiscoveryInfo.java	2010-03-11 20:56:29.795784348 +0100
@@ -20,7 +20,7 @@
  */
 
 
-
+package airporthostmon;
 
 import java.util.*;
 
Index: airport-utils-2/hostmon/AirportExtremeHostInfoRetriever.java
===================================================================
--- airport-utils-2.orig/hostmon/AirportExtremeHostInfoRetriever.java	2010-03-11 20:55:55.917753422 +0100
+++ airport-utils-2/hostmon/AirportExtremeHostInfoRetriever.java	2010-03-11 20:56:29.837402701 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airporthostmon;
 
 import snmp.*;
 import java.io.*;
Index: airport-utils-2/hostmon/AirportInfoRecord.java
===================================================================
--- airport-utils-2.orig/hostmon/AirportInfoRecord.java	2010-03-11 20:55:55.926753887 +0100
+++ airport-utils-2/hostmon/AirportInfoRecord.java	2010-03-11 20:56:29.854154021 +0100
@@ -20,7 +20,7 @@
  */
 
 
-
+package airporthostmon;
 
 import java.math.*;
 import java.util.*;
Index: airport-utils-2/hostmon/ArpInfo.java
===================================================================
--- airport-utils-2.orig/hostmon/ArpInfo.java	2010-03-11 20:55:55.935753453 +0100
+++ airport-utils-2/hostmon/ArpInfo.java	2010-03-11 20:56:29.870909696 +0100
@@ -20,7 +20,7 @@
  */
 
 
-
+package airporthostmon;
 
 
 public class ArpInfo
Index: airport-utils-2/hostmon/ArpInfoTreeMap.java
===================================================================
--- airport-utils-2.orig/hostmon/ArpInfoTreeMap.java	2010-03-11 20:55:55.944753485 +0100
+++ airport-utils-2/hostmon/ArpInfoTreeMap.java	2010-03-11 20:56:29.887752249 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airporthostmon;
 
 import java.util.*;
 import java.math.*;
Index: airport-utils-2/hostmon/EmailMessage.java
===================================================================
--- airport-utils-2.orig/hostmon/EmailMessage.java	2010-03-11 20:55:55.953753291 +0100
+++ airport-utils-2/hostmon/EmailMessage.java	2010-03-11 20:56:29.904419884 +0100
@@ -19,6 +19,7 @@
  *
  */
  
+package airporthostmon;
  
  /* EmailMessage class
 
Index: airport-utils-2/hostmon/GraphiteAirportHostInfoRetriever.java
===================================================================
--- airport-utils-2.orig/hostmon/GraphiteAirportHostInfoRetriever.java	2010-03-11 20:55:55.962753531 +0100
+++ airport-utils-2/hostmon/GraphiteAirportHostInfoRetriever.java	2010-03-11 20:56:29.946165992 +0100
@@ -19,6 +19,7 @@
  *
  */
 
+package airporthostmon;
 
 import snmp.*;
 import java.io.*;
Index: airport-utils-2/hostmon/HostInfo.java
===================================================================
--- airport-utils-2.orig/hostmon/HostInfo.java	2010-03-11 20:55:55.971752876 +0100
+++ airport-utils-2/hostmon/HostInfo.java	2010-03-11 20:56:29.962920790 +0100
@@ -19,6 +19,8 @@
  *
  */
 
+package airporthostmon;
+
 import java.util.*;
 
 public class HostInfo
Index: airport-utils-2/hostmon/HostInfoBeepAlerter.java
===================================================================
--- airport-utils-2.orig/hostmon/HostInfoBeepAlerter.java	2010-03-11 20:55:55.980753586 +0100
+++ airport-utils-2/hostmon/HostInfoBeepAlerter.java	2010-03-11 20:56:29.979752327 +0100
@@ -19,7 +19,7 @@
  *
  */
 
- 
+package airporthostmon; 
  
  import java.io.*;
  import java.util.*;
Index: airport-utils-2/hostmon/HostInfoChangeEmailer.java
===================================================================
--- airport-utils-2.orig/hostmon/HostInfoChangeEmailer.java	2010-03-11 20:55:55.989758653 +0100
+++ airport-utils-2/hostmon/HostInfoChangeEmailer.java	2010-03-11 20:56:29.996428812 +0100
@@ -18,7 +18,8 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
  */
- 
+
+package airporthostmon; 
  
 import java.io.*;
 import java.util.*;
Index: airport-utils-2/hostmon/HostInfoChangeException.java
===================================================================
--- airport-utils-2.orig/hostmon/HostInfoChangeException.java	2010-03-11 20:55:55.999753412 +0100
+++ airport-utils-2/hostmon/HostInfoChangeException.java	2010-03-11 20:56:30.013186647 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airporthostmon;
 
 
 /** 
Index: airport-utils-2/hostmon/HostInfoChangeListener.java
===================================================================
--- airport-utils-2.orig/hostmon/HostInfoChangeListener.java	2010-03-11 20:55:56.008753679 +0100
+++ airport-utils-2/hostmon/HostInfoChangeListener.java	2010-03-11 20:56:30.054931534 +0100
@@ -20,7 +20,7 @@
  */
 
  
-
+package airporthostmon;
 
 
 public interface HostInfoChangeListener
Index: airport-utils-2/hostmon/HostInfoLogger.java
===================================================================
--- airport-utils-2.orig/hostmon/HostInfoLogger.java	2010-03-11 20:55:56.017753557 +0100
+++ airport-utils-2/hostmon/HostInfoLogger.java	2010-03-11 20:56:30.071752180 +0100
@@ -19,7 +19,7 @@
  *
  */
 
- 
+package airporthostmon; 
  
  import java.io.*;
  import java.util.*;
Index: airport-utils-2/hostmon/HostInfoRetriever.java
===================================================================
--- airport-utils-2.orig/hostmon/HostInfoRetriever.java	2010-03-11 20:56:29.481753088 +0100
+++ airport-utils-2/hostmon/HostInfoRetriever.java	2010-03-11 20:56:30.088438433 +0100
@@ -19,6 +19,7 @@
  *
  */
 
+package airporthostmon;
 
 import snmp.*;
 
Index: airport-utils-2/hostmon/HostInfoTable.java
===================================================================
--- airport-utils-2.orig/hostmon/HostInfoTable.java	2010-03-11 20:55:56.035753211 +0100
+++ airport-utils-2/hostmon/HostInfoTable.java	2010-03-11 20:56:30.105194288 +0100
@@ -19,7 +19,7 @@
  *
  */
 
- 
+package airporthostmon; 
  
 import java.awt.*;
 import java.awt.event.*;
Index: airport-utils-2/hostmon/HostInfoTreeMap.java
===================================================================
--- airport-utils-2.orig/hostmon/HostInfoTreeMap.java	2010-03-11 20:55:56.044753490 +0100
+++ airport-utils-2/hostmon/HostInfoTreeMap.java	2010-03-11 20:56:30.146940014 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airporthostmon;
 
 import java.util.*;
 import java.math.*;
Index: airport-utils-2/hostmon/HostMonitor.java
===================================================================
--- airport-utils-2.orig/hostmon/HostMonitor.java	2010-03-11 20:55:56.053753096 +0100
+++ airport-utils-2/hostmon/HostMonitor.java	2010-03-11 20:56:30.163752075 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airporthostmon;
 
 
 import java.util.*;
Index: airport-utils-2/hostmon/InfoPanel.java
===================================================================
--- airport-utils-2.orig/hostmon/InfoPanel.java	2010-03-11 20:55:56.062753179 +0100
+++ airport-utils-2/hostmon/InfoPanel.java	2010-03-11 20:56:30.180447753 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airporthostmon;
 
 import java.awt.*;
 import java.awt.event.*;
Index: airport-utils-2/hostmon/JDataButton.java
===================================================================
--- airport-utils-2.orig/hostmon/JDataButton.java	2010-03-11 20:55:56.071752984 +0100
+++ airport-utils-2/hostmon/JDataButton.java	2010-03-11 20:56:30.197203127 +0100
@@ -19,6 +19,7 @@
  *
  */
 
+package airporthostmon;
 
 import javax.swing.*;
 
Index: airport-utils-2/hostmon/JTextAreaWriter.java
===================================================================
--- airport-utils-2.orig/hostmon/JTextAreaWriter.java	2010-03-11 20:55:56.080753887 +0100
+++ airport-utils-2/hostmon/JTextAreaWriter.java	2010-03-11 20:56:30.238948668 +0100
@@ -18,7 +18,8 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
  */
- 
+
+package airporthostmon; 
  
 import java.io.*;
 import javax.swing.text.*;
Index: airport-utils-2/hostmon/KnownHostDialog.java
===================================================================
--- airport-utils-2.orig/hostmon/KnownHostDialog.java	2010-03-11 20:55:56.089753237 +0100
+++ airport-utils-2/hostmon/KnownHostDialog.java	2010-03-11 20:56:30.255752012 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airporthostmon;
 
 
 import java.awt.*;
Index: airport-utils-2/hostmon/KnownHostInfo.java
===================================================================
--- airport-utils-2.orig/hostmon/KnownHostInfo.java	2010-03-11 20:55:56.098753525 +0100
+++ airport-utils-2/hostmon/KnownHostInfo.java	2010-03-11 20:56:30.272460697 +0100
@@ -18,7 +18,8 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
  */
- 
+
+package airporthostmon; 
 
 import java.util.*;
 import java.io.*;
Index: airport-utils-2/hostmon/KnownHostTable.java
===================================================================
--- airport-utils-2.orig/hostmon/KnownHostTable.java	2010-03-11 20:56:29.445753934 +0100
+++ airport-utils-2/hostmon/KnownHostTable.java	2010-03-11 20:56:30.289212268 +0100
@@ -19,7 +19,7 @@
  *
  */
 
- 
+package airporthostmon; 
  
 import java.awt.*;
 import java.awt.event.*;
Index: airport-utils-2/hostmon/MacAddress.java
===================================================================
--- airport-utils-2.orig/hostmon/MacAddress.java	2010-03-11 20:55:56.116753227 +0100
+++ airport-utils-2/hostmon/MacAddress.java	2010-03-11 20:56:30.330963245 +0100
@@ -19,6 +19,8 @@
  *
  */
 
+package airporthostmon;
+
 import java.util.*;
 import java.io.*;
 
Index: airport-utils-2/hostmon/MailerThread.java
===================================================================
--- airport-utils-2.orig/hostmon/MailerThread.java	2010-03-11 20:55:56.125753467 +0100
+++ airport-utils-2/hostmon/MailerThread.java	2010-03-11 20:56:30.347751945 +0100
@@ -18,7 +18,8 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
  */
- 
+
+package airporthostmon; 
 
 import java.io.*;
 import java.util.*;
Index: airport-utils-2/hostmon/POPWeasel.java
===================================================================
--- airport-utils-2.orig/hostmon/POPWeasel.java	2010-03-11 20:55:56.134753489 +0100
+++ airport-utils-2/hostmon/POPWeasel.java	2010-03-11 20:56:30.364472346 +0100
@@ -18,6 +18,8 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
  */
+
+package airporthostmon;
  
  /* class POPWeasel
 
Index: airport-utils-2/hostmon/Preferences.java
===================================================================
--- airport-utils-2.orig/hostmon/Preferences.java	2010-03-11 20:55:56.143753625 +0100
+++ airport-utils-2/hostmon/Preferences.java	2010-03-11 20:56:30.406217688 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airporthostmon;
 
 import java.io.*;
 import java.net.*;
Index: airport-utils-2/hostmon/Preferences2.java
===================================================================
--- airport-utils-2.orig/hostmon/Preferences2.java	2010-03-11 20:55:56.152753091 +0100
+++ airport-utils-2/hostmon/Preferences2.java	2010-03-11 20:56:30.422972877 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airporthostmon;
 
 import java.io.*;
 import java.net.*;
Index: airport-utils-2/hostmon/PreferencesDialog.java
===================================================================
--- airport-utils-2.orig/hostmon/PreferencesDialog.java	2010-03-11 20:55:56.161753586 +0100
+++ airport-utils-2/hostmon/PreferencesDialog.java	2010-03-11 20:56:30.439752957 +0100
@@ -20,7 +20,7 @@
  */
 
  
- 
+package airporthostmon; 
 
 import java.awt.*;
 import javax.swing.*;
Index: airport-utils-2/hostmon/RarpInfoTreeMap.java
===================================================================
--- airport-utils-2.orig/hostmon/RarpInfoTreeMap.java	2010-03-11 20:55:56.170753645 +0100
+++ airport-utils-2/hostmon/RarpInfoTreeMap.java	2010-03-11 20:56:30.464810346 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airporthostmon;
 
 import java.util.*;
 import java.math.*;
Index: airport-utils-2/hostmon/SnowAirportHostInfoRetriever.java
===================================================================
--- airport-utils-2.orig/hostmon/SnowAirportHostInfoRetriever.java	2010-03-11 20:55:56.179753506 +0100
+++ airport-utils-2/hostmon/SnowAirportHostInfoRetriever.java	2010-03-11 20:56:30.506556661 +0100
@@ -19,6 +19,7 @@
  *
  */
 
+package airporthostmon;
 
 import snmp.*;
 
Index: airport-utils-2/hostmon/ValueFormatException.java
===================================================================
--- airport-utils-2.orig/hostmon/ValueFormatException.java	2010-03-11 20:55:56.191753335 +0100
+++ airport-utils-2/hostmon/ValueFormatException.java	2010-03-11 20:56:30.523311576 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airporthostmon;
 
 
 /** 
